README.md

```diff
@@ -1,8 +1,10 @@
-# New Relic Infrastructure Integration for VMware vSphere
+[![Community Project header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Project.png)](https://opensource.newrelic.com/oss-category/#community-project)
 
-New Relic Infrastructure Integration for VMware vShpere captures critical summary metrics and inventory data by connecting to the vCenter or to an ESXi Host. For more information, see [the official New Relic docs](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/vmware-vsphere-monitoring-integration).
+# New Relic VMware vShpere integration [![Known Vulnerabilities](https://snyk.io/test/github/newrelic/nri-vsphere/badge.svg?targetFile=vendor/vendor.json)](https://snyk.io/test/github/newrelic/nri-vsphere?targetFile=vendor/vendor.json)
```

Comment:
@ardias Is this supposed to be a button? I'd put it in the line below the title, not at the same level as the title.

Reply:
Not a button, but an image (badge).
Makes sense to move below.